### PR TITLE
social-auth-core <4.0.3 constraint

### PR DESCRIPTION
**Pull request by Tj-Tracy**
version pin ticket: https://openedx.atlassian.net/browse/MICROBA-929

### CODE REVIEW

File: requirements/all.txt [hunk 1]
```diff
@@ -559,8 +578,6 @@ pytz==2020.5
     #   -r requirements/production.txt
     #   django
     #   django-ses
-pywatchman==1.4.1 ; "linux" in sys_platform
```
[the remark this answers is not in the document]
Comment (Tj-Tracy) [reply requirements/all.txt:562]: I see, should I be running `make upgrade` from within the container then?

File: requirements/all.txt [hunk 2]
```diff
@@ -312,7 +316,7 @@ gitpython==3.1.12
     #   -r requirements/dev.txt
     #   -r requirements/production.txt
     #   transifex-client
-greenlet==0.4.17
+greenlet==1.0.0
```
[the remark this answers is not in the document]
Comment (Tj-Tracy) [reply requirements/all.txt:319]: no breaking changes were called out in the changelog, as far as I can tell.

File: requirements/all.txt [hunk 3]
```diff
@@ -675,8 +692,9 @@ social-auth-app-django==4.0.0
     #   -r requirements/dev.txt
     #   -r requirements/production.txt
     #   edx-auth-backends
-social-auth-core==3.3.3
+social-auth-core==4.0.2
```
[the remark this answers is not in the document]
Comment (Tj-Tracy) [reply requirements/all.txt:695]: Yeah, I looked through them while searching for the issue with the build. No breaking changes were called out, as far as I can tell.

File: requirements/all.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (Tj-Tracy) [reply requirements/all.txt]: this was fixed after I ran it from the container.

**Review by crice100 — APPROVED** [review]

**Pull request merged**